tests/test_datasets.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import pandas as pd
+import torchvision.transforms as T
+from quickvision.datasets.classification import CSVDataset
+
+df = pd.read_csv('tests/assets/csv_dataset.csv')
+data_dir = 'tests/assets/'
+tfms = T.Compose([
+    T.ToTensor(),
+])
+
+
+class DatasetsTester(unittest.TestCase):
```

Comment:
Maybe call it `ClassificationDatasetsTester`?  coz we would have similar for Detection etc.

Reply:
I will call it `CSVSingleLabelDatasetTester`